build.gradle.kts

```diff
@@ -11,6 +11,14 @@ version = "0.0.2"
 
 repositories {
     mavenCentral()
+    maven {
```

Comment:
Tror vi kan benytte oss av:
 maven {
        url = uri("https://github-package-registry-mirror.gc.nav.no/cached/maven-release")
    }
Da slipper vi å ha x-access-token lokalt og på CI/CD

Reply:
Der det er muligt, og ikkje for mye jobb å få til, tenker eg at det er fint å ungå.
Da blir det litt raskere og enkelete å komme igang med utsjekk av koden.
Dersom man skal ha tokens, må det beskrives eks i README.md hvilken det er og hvordan man skal sette dei opp tenker eg

Reply:
Sikkert lurt, har pleid å ha aksess token liggende lokalt i en fil som er i .gitignore tidligere. Men om det du foreslår funker er det jo bedre